### ISSUE TITLE
Value

### ISSUE THREAD

**Issue by wkl000** [AustinApple/modified_chemvae#1]

**Comment by wkl000** [comment]
Hello, when I change the data file. I use the original encoder.h5 and decoder.h5, and add "do_prop_pred" to the exp_property_training.json. Then I run the train_prop.py. But I have following error:

Using hyper-parameters:
name                      - 70W_MP_property_training
MAX_LEN                   - 120         
data_file                 - datalungnci23 copy.csv
char_file                 - datalungnci23 copy.json
encoder_weights_file      - zinc_encoder.h5
decoder_weights_file      - zinc_decoder.h5
prop_pred_weights_file    - 70W_MP_prop_pred.h5
reg_prop_tasks            - ['GIPRCNT'] 
test_idx_file             - test_idx.npy
history_file              - history     
do_prop_pred              - False       
do_prop_pred_only         - True        
TRAIN_MODEL               - True        
ENC_DEC_TEST              - False       
PADDING                   - right       
RAND_SEED                 - 42          
epochs                    - 200         
vae_annealer_start        - 29          
dropout_rate_mid          - 0.08283292970479479
anneal_sigmod_slope       - 0.5106654305791392
recurrent_dim             - 488         
hidden_dim                - 196         
tgru_dropout              - 0.19617749608323892
hg_growth_factor          - 1.2281884874932403
middle_layer              - 1           
prop_hidden_dim           - 67          
batch_size                - 256         
prop_pred_depth           - 3           
lr                        - 0.00045619868229310396
prop_pred_dropout         - 0.15694573998898703
prop_growth_factor        - 0.9902834073131418
momentum                  - 0.9902764103622574
reload_model              - True        
rest of parameters are set as default
All params: {'reload_model': True, 'prev_epochs': 0, 'batch_size': 256, 'epochs': 200, 'val_split': 0.1, 'loss': 'categorical_crossentropy', 'batchnorm_conv': True, 'conv_activation': 'tanh', 'conv_depth': 4, 'conv_dim_depth': 8, 'conv_dim_width': 8, 'conv_d_growth_factor': 1.15875438383, 'conv_w_growth_factor': 1.1758149644, 'gru_depth': 4, 'rnn_activation': 'tanh', 'recurrent_dim': 488, 'do_tgru': True, 'terminal_GRU_implementation': 0, 'tgru_dropout': 0.19617749608323892, 'temperature': 1.0, 'hg_growth_factor': 1.2281884874

If I have a new dataset, I want to use the original ZINC data to train the VAE and then use my own data with one property to do the prediction, how can I use your code to train? It will be great if you could provide a procedure.

Thank you.
